forem/forem
# Update nokogiri from 1.13.1 to 1.13.2

## What type of PR is this? (check all applicable)


## Description

Fixes CVE-2021-30560 / [GHSA-fq42-c5rg-92c2](https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-fq42-c5rg-92c2) and related bundle-audit failure blocking builds.

## Related Tickets & Documents

https://app.travis-ci.com/github/forem/forem/jobs/560614464 failure shows here

https://github.com/sparklemotion/nokogiri/releases/tag/v1.13.2 changelog includes only the library update

## QA Instructions, Screenshots, Recordings

```
   bundle exec bundle-audit check --update
```

Should report no vulnerabilities found.

### UI accessibility concerns?

None

## Added/updated tests?

- [x] No, and this is why: patch

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [x] This change does not need to be communicated, and this is why not: security patch

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
